sale_price_origin/models/sale.py

```diff
@@ -0,0 +1,31 @@
+# coding: utf-8
+# © 2017 David BEAL @ Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = 'sale.order.line'
+
+    price_origin = fields.Many2one(
+        comodel_name='product.pricelist.item', string="Price Origin",
+        compute='_compute_price_origin', readonly=True, ondelete='cascade',
+        help="Define which product pricelist item has been used "
+             "to compute the price unit")
```

Comment:
Is there a good reason not to set store=True ?
for sale order with a lot a line, I guess it will generate performance problem.